packages/hdwallet/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@truffle/hdwallet",
+  "description": "HD Wallet helpers for Truffle",
+  "license": "MIT",
+  "author": "david <david@trufflesuite.com>",
+  "homepage": "https://github.com/trufflesuite/truffle/tree/master/packages/hdwallet#readme",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/trufflesuite/truffle.git",
+    "directory": "packages/hdwallet"
+  },
+  "bugs": {
+    "url": "https://github.com/trufflesuite/truffle/issues"
+  },
+  "version": "0.1.0-0",
+  "main": "dist/src/index.js",
+  "scripts": {
+    "build": "ttsc",
+    "prepare": "yarn build",
+    "test": "exit 0"
```

Comment:
@lsqproduction discovered an issue with Node 18, on M1 OS X which is triggered by running `yarn test` from the repo root. This can be simulated by scoping `@truffle/hdwallet` via `$(yarn bin)/lerna run test --stream --concurrency=1 --scope @truffle/hdwallet -- --colors`. 

One suggestion is to remove the `test` script from `@truffle/hdwallet` as the test script does nothing.

# error output
```
$ $(yarn bin)/lerna run test --stream --concurrency=1 --scope @truffle/hdwallet -- --colors
lerna notice cli v4.0.0
lerna info versioning independent
lerna notice filter including "@truffle/hdwallet"
lerna info filter [ '@truffle/hdwallet' ]
lerna info Executing command in 1 package: "yarn run test --colors"
yarn run v1.22.19
$ exit 0 --colors
@truffle/hdwallet: /bin/sh: line 0: exit: too many arguments
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
lerna ERR! yarn run test --colors exited 1 in '@truffle/hdwallet'
```

Reply:
This issue was caused by `hdwallet` not properly invoking its tests. It has been [fixed in this commit](https://github.com/trufflesuite/truffle/pull/5543/commits/7769645ba4c7d4fcd3f4a0262cbccf38a9b127c2). Thanks for bringing attention to this issue @lsqproduction. We all missed it!